project/Build.scala

```diff
@@ -349,6 +351,12 @@ object DottyInjectedPlugin extends AutoPlugin {
      )
    )
 
+   lazy val `dotty-core` = project
```

Comment:
I think the root project should be the one that aggregates the others (like in Ensime: https://github.com/ensime/ensime-server/blob/2.0/project/EnsimeBuild.scala#L243) but I don't know if that requires doing something special to get `compile`, `run`, etc to do the correct thing